### ISSUE TITLE
!pip install --upgrade tensorlayer==1.11.0 crashes runtime ( it was working last week)

### ISSUE THREAD

**Issue by dpredie** [googlecolab/colabtools#2601]
**Describe the current behavior**
```
%tensorflow_version 1.x
!pip install --upgrade tensorlayer==1.11.0

```
this crashes the runtime repeatedly, it was working fine last week

**What web browser you are using**
Edge

**Additional context**
(https://colab.research.google.com/github/dpredie/SingleHDR/blob/master/SingleHDR.ipynb)

**Comment by craigcitro** [comment]
Thanks for the bug report!

The situation here is as follows:
* colab is proactively importing pandas as part of kernel startup
* installing `tensorlayer` is installing an old version of numpy
* that numpy version is incompatible with the pandas version we have installed by default, leading to failures on restart

I've submitted a fix for the first part (we don't need to proactively import pandas), which avoids the crash looping, but with a messy error. I'll update this bug once that rolls out.

Even before that, though, we can get you unblocked: you'll want to also reinstall an old version of pandas (eg `!pip install --upgrade pandas==1.0.5` or `!pip install --upgrade pandas==1.1.5`) alongside `tensorlayer` to work with the older numpy.
